content/influxdb/v2.2/write-data/best-practices/resolve-high-cardinality.md

```diff
@@ -9,7 +9,8 @@ menu:
     parent: write-best-practices
 ---
 
-If reads and writes to InfluxDB have started to slow down, high [series cardinality](/influxdb/v2.2/reference/glossary/#series-cardinality) (too many series) may be causing memory issues.
+If reads and writes to InfluxDB have started to slow down, high [series cardinality](/influxdb/v2.1/reference/glossary/#series-cardinality) (too many series) may be causing memory issues. {{% cloud-only %}}Cardinality can also cause writes to fail if it exceeds your [plan’s adjustable service quota](/influxdb/cloud/account-management/limits/).{{% /cloud-only %}}
```

Comment:
```suggestion
If reads and writes to InfluxDB have started to slow down, high [series cardinality](/influxdb/v2.1/reference/glossary/#series-cardinality) might be causing memory issues. {{% cloud-only %}}Cardinality can also cause writes to fail if it exceeds your [plan’s adjustable service quota](/influxdb/cloud/account-management/limits/).{{% /cloud-only %}}
```

Reply:
Thanks. Leaving as-is b/c I think Kelly might have originally suggested it.